components/more_channels/more_channels.jsx

```diff
@@ -172,6 +172,13 @@ export default class MoreChannels extends React.Component {
             </TeamPermissionGate>
         );
 
+        let channelsList = [];
+        if (search) {
+            channelsList = searchedChannels.filter((c) => c.delete_at === 0);
+        } else {
+            channelsList = channels.filter((c) => c.delete_at === 0);
+        }
```

Comment:
Should the filtering be done outside the render function?  Like,
-  on `searchedChannels` [before setting the state](https://github.com/mattermost/mattermost-webapp/blob/6c348ef6e830490698a616a4ccb3aca44067a7ef/components/more_channels/more_channels.jsx#L114).
- on `channels` [before passing as props](https://github.com/mattermost/mattermost-webapp/blob/v5.2.0-rc2/components/more_channels/index.js#L17)

Reply:
Good point

Reply:
Done! PTAL